apps/rxjs/38-catch-error/src/app/app.component.ts

```diff
@@ -42,19 +43,22 @@ export class AppComponent implements OnInit {
       .pipe(
         map(() => this.input),
         concatMap((value) =>
-          this.http.get(`https://jsonplaceholder.typicode.com/${value}/1`),
+          this.http.get(`https://jsonplaceholder.typicode.com/${value}/1`).pipe(
+            tapResponse({
```

Comment:
that's true, you can use tapResponse even outside of a component-store application, 😅

Reply:
Yeah, I first came across is in a signalStore example (I'd never used component-store).